src/Client.js

```diff
@@ -125,6 +126,16 @@ Client.prototype._execute = function (action, path, data, query) {
       response.text, responseObject, response.status, response.header,
       startTime, endTime);
 
+    if ('x-last-seen-txn' in response.header) {
+        var time = parseInt(response.header['x-last-seen-txn'], 10);
```

Comment:
that could be an issue on javascript as it cannot represent a java long value 😞

Reply:
Yep.

Reply:
I hate this language.

Reply:
Idk... split the string in half and compare both sides? _(runs aways really quickly)_

We might need to pick a library for this....

Reply:
Seems like `BigInt` will be part of js standard some day in the future, nodejs and chrome already support it, but safari don't

Reply:
I think this is safe... 2^53 micros since the epoch is 11:47:34 pm UTC Tuesday, June 5, 2255. That seems sufficient...

Reply:
Who would say math was gonna give us a simpler solution, hum?!